app/views/profiles/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="container justify-content-center text-center my-5">
+  <h3 class="text-primary text-center">Profile</h3>
+  <div class="w-50 m-auto"> 
+    <table class="table table-bordered table-hover table-striped text-center mt-4">
+      <thead class="table-dark">
+        <tr>
+          <th scope="col">Username</th>
+          <th scope="col">Address</th>
+        </tr>
+      </thead>
+      
+      <tbody>
+        <tr>
+          <td><%= @current_user.username %></td>
+          <td><%= @address.address_line %>, <%= @address.city %>, <%= @address.state %></td>
+        </tr>
+      </tbody>
+    </table>
+    <td><%= link_to "Edit Address", edit_address_path(@address.id), :class => "btn btn-warning" %></td>
+    <hr>
+  </div>
+  <h3 class="text-success text-center mt-5">You have Enrolled Following Events</h3>
+  <div class="w-50 m-auto"> 
+    <table class="table table-bordered table-hover table-striped text-center mt-4">
+      <thead class="table-dark">
+        <tr>
+          <th scope="col">Name</th>
+          <th scope="col">Category</th>
+          <th scope="col">Description</th>
+          <th scope="col">date</th>
+          <th scope="col">Show</th>
+          <th scope="col">Enroll</th>
+        </tr>
+      </thead>
+      
+      <tbody>
+        <% @events.reverse_each do |event| %>
```

Comment:
Why using reverse_each ?

Reply:
To show events in decreasing order. it is required in index page, but it in not required here. so i replace it with each.